docs/dev/06-git-commit-msg.md

```diff
@@ -55,16 +56,9 @@ The `<scope>` can be empty (e.g. if the change is a global or difficult
 to assign to a single component), in which case the parentheses are
 omitted. In smaller projects such as Karma plugins, the `<scope>` is empty.
 
-
 ## Message body
-* uses the imperative, present tense: “change” not “changed” nor “changes”
-* includes motivation for the change and contrasts with previous behavior
-
-For more info about message body, see:
-
-* https://365git.tumblr.com/post/3308646748/writing-git-commit-messages
-* https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
 
+Just as in the `<subject>`, use the imperative, present tense: "change" not "changed" nor "changes". The body should include the motivation for the change and contrast this with previous behavior. Message body should include motivation for the change and contrasts with previous behavior.
```

Comment:
Did we copy this and the other paragraphs from Angular's repo?

Reply:
I guess the original prose was copied/inspired by the AngularJS commit message conventions. This particular paragraph I've copied from https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit, yes.

PS Just noticed that I accidentally added the same sentence twice. I'll push a fix shortly.

Reply:
SG.